Transbank/PatpassRest/Common/RequestService.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Transbank.Common;
+using Transbank.Exceptions;
+
+namespace Transbank.Patpass.Common
+{
+    public class RequestService
```

Comment:
Why did not you create an abstract class with the code of method `Perform` in the namespace `Transabank .Common` (as I could see this method are identical on both places) and with the abstract method `AddRequiredHeaders` so you could then create two different specialized classes for Webpay and for PatpassComercio that extends this abstract class and implements the method with the corresponding headers.